.clippy.toml

```diff
@@ -1,2 +1,2 @@
 msrv = "1.70.0"
-cognitive-complexity-threshold = 10
+cognitive-complexity-threshold = 24
```

Comment:
Wait this was super low all this time??? No wonder it was so annoying. Let's just remove this line; the default value is 25 according to [the documentation](https://doc.rust-lang.org/nightly/clippy/lint_configuration.html#cognitive-complexity-threshold)

Reply:
@sylvestre I see that you introduced the lower value. I think 10 is a bit too low, but I'm happy to compromise on something inbetween as well.

Reply:
sure, it was to identify too long/complex functions. I am happy where we are now with 25
thanks